processor/schemaprocessor/README.md

```diff
@@ -0,0 +1,38 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Processor_ is used to convert existing telemetry data or signals to a version of the semantic convention defined as part of the configuration.
+The processor works by using a set of target schema URLs that are used to match incoming signal.
+On a match, the processor will fetch the schema translation file (if not cached) set by the incoming signal and apply transformations
+required to export as the target semantic convention version.
+
+Furthemore, it is also possible for organisations and vendors to publish their own semantic convention and be used by this processor, 
+be sure to follow [schema overview](https://opentelemetry.io/docs/reference/specification/schemas/overview/) for all the details.
+
+## Caching Schema Translation Files
+
+In order to improve effeicency of the processor, the `precache` option allows the processor to start downloading and preparing
```

Comment:
Is it too late to rename `precache` to `prefetch`? I think it's more widely used for similar features elsewhere.

Reply:
Makes sense to me :D